modules/message.js

```diff
@@ -1974,7 +1976,9 @@ let PostStreamingFixesMixIn = {
       let _log = function () {
         let t1 = Date.now();
         let delta = t1 - t;
-        let args = [x for each ([, x] in Iterator(arguments))];
```

Comment:
Eeeeew. I don't know what this code is doing, but I'd rather get rid of `_log` entirely. Who wrote that? ;-)